plugins/eslint-plugin-aws-local/index.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import NoOnlyInTests from './lib/rules/no-only-in-tests'
+
+const rules = {
+    'no-only-in-tests': NoOnlyInTests,
+}
+
+export { rules }
```

Comment:
is prettier setup on your machine? might need to configure your editor to ensure the file ends with a newline.

not sure why prettier isn't catching this.

Reply:
needed to add it to the prettier command- nice catch